`ElementHandle.Press()` panic: Invalid 'text' parameter

On current `main` (917b83e) the following test:

```go
func TestElementHandlePress(t *testing.T) {
	tb := newTestBrowser(t)
	p := tb.NewPage(nil)
	p.SetContent(`<input type="text">`, nil)
	el := p.Query("input")
	el.Press("Backspace", nil)
}
```

... panics with:

```
--- FAIL: TestElementHandlePress (0.34s)
panic: cannot handle element key ("Backspace") press: cannot do key down: cannot execute dispatch key event down: Invalid 'text' parameter (-32602) [recovered]
        panic: cannot handle element key ("Backspace") press: cannot do key down: cannot execute dispatch key event down: Invalid 'text' parameter (-32602)

goroutine 9 [running]:
testing.tRunner.func1.2({0x17ac1e0, 0xc0001c3530})
        /home/ivan/.local/share/go/src/testing/testing.go:1209 +0x36c
testing.tRunner.func1()
        /home/ivan/.local/share/go/src/testing/testing.go:1212 +0x3b6
panic({0x17ac1e0, 0xc0001c3530})
        /home/ivan/.local/share/go/src/runtime/panic.go:1047 +0x266
go.k6.io/k6/js/common.Throw(0xc0006a5c88, {0x19c9680, 0xc000121ad0})
        /home/ivan/Projects/grafana/xk6-browser/vendor/go.k6.io/k6/js/common/util.go:36 +0x79
github.com/grafana/xk6-browser/common.k6Throw({0x19e6a60, 0xc000692060}, {0x1810dc4, 0x28}, {0xc0006a5e10, 0x2, 0x2})
        /home/ivan/Projects/grafana/xk6-browser/common/helpers.go:214 +0x22f
github.com/grafana/xk6-browser/common.(*ElementHandle).Press(0xc00017aa80, {0x17e6cee, 0x5}, {0x0, 0x0})
        /home/ivan/Projects/grafana/xk6-browser/common/element_handle.go:1169 +0x3d6
github.com/grafana/xk6-browser/tests.TestElementHandlePress(0x0)
        /home/ivan/Projects/grafana/xk6-browser/tests/element_handle_test.go:40 +0xd7
testing.tRunner(0xc000183380, 0x183cbd0)
        /home/ivan/.local/share/go/src/testing/testing.go:1259 +0x230
created by testing.(*T).Run
        /home/ivan/.local/share/go/src/testing/testing.go:1306 +0x727
FAIL    github.com/grafana/xk6-browser/tests    0.411s
```

From initial investigation, it seems like `text` shouldn't be set to `key` in some cases, as done here:
https://github.com/grafana/xk6-browser/blob/917b83ef14b3e2075f92c092163c47e359883a6a/common/keyboard.go#L198-L201

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -15,6 +15,11 @@ issues:
   # When true it disables revive's comment warnings, and we want them enabled.
   exclude-use-default: false
 
+  exclude-rules:
+    - linters:
+        - funlen
+      source: "^func Test"
+
 linters:
   disable-all: true
   enable:

--- a/common/keyboard.go
+++ b/common/keyboard.go
@@ -197,6 +197,8 @@ func (k *Keyboard) keyDefinitionFromKey(key keyboardlayout.KeyInput) keyboardlay
 	var keyDef keyboardlayout.KeyDefinition
 	if srcKeyDef.Key != "" {
 		keyDef.Key = srcKeyDef.Key
+	}
+	if len(srcKeyDef.Key) == 1 {
 		keyDef.Text = srcKeyDef.Key
 	}
 	if shift != 0 && srcKeyDef.ShiftKeyCode != 0 {

--- a/tests/keyboard_test.go
+++ b/tests/keyboard_test.go
@@ -1,0 +1,64 @@
+package tests
+
+import (
+	_ "embed"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/xk6-browser/common"
+	"github.com/grafana/xk6-browser/keyboardlayout"
+)
+
+func TestKeyboardPress(t *testing.T) {
+	tb := newTestBrowser(t)
+
+	t.Run("all_keys", func(t *testing.T) {
+		p := tb.NewPage(nil)
+		cp, ok := p.(*common.Page)
+		require.True(t, ok)
+		kb := cp.Keyboard
+		layout := keyboardlayout.GetKeyboardLayout("us")
+
+		assert.NotPanics(t, func() {
+			for k := range layout.Keys {
+				kb.Press(string(k), nil)
+			}
+		})
+	})
+
+	t.Run("backspace", func(t *testing.T) {
+		p := tb.NewPage(nil)
+		cp, ok := p.(*common.Page)
+		require.True(t, ok)
+		kb := cp.Keyboard
+
+		p.SetContent(`<input>`, nil)
+		el := p.Query("input")
+		p.Focus("input", nil)
+
+		kb.Type("Hello World!", nil)
+		require.Equal(t, "Hello World!", el.InputValue(nil))
+
+		kb.Press("Backspace", nil)
+		assert.Equal(t, "Hello World", el.InputValue(nil))
+	})
+
+	t.Run("newline", func(t *testing.T) {
+		p := tb.NewPage(nil)
+		cp, ok := p.(*common.Page)
+		require.True(t, ok)
+		kb := cp.Keyboard
+
+		p.SetContent(`<textarea>`, nil)
+		el := p.Query("textarea")
+		p.Focus("textarea", nil)
+
+		kb.Type("Hello", nil)
+		kb.Press("Enter", nil)
+		kb.Press("Enter", nil)
+		kb.Type("World!", nil)
+		assert.Equal(t, "Hello\n\nWorld!", el.InputValue(nil))
+	})
+}

--- a/tests/keyboard_test.go
+++ b/tests/keyboard_test.go
@@ -2,6 +2,7 @@ package tests
 
 import (
 	_ "embed"
+	"runtime"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
@@ -45,6 +46,32 @@ func TestKeyboardPress(t *testing.T) {
 		assert.Equal(t, "Hello World", el.InputValue(nil))
 	})
 
+	t.Run("combo", func(t *testing.T) {
+		t.Skip("FIXME") // See https://github.com/grafana/xk6-browser/issues/285
+		p := tb.NewPage(nil)
+		cp, ok := p.(*common.Page)
+		require.True(t, ok)
+		kb := cp.Keyboard
+
+		p.SetContent(`<input>`, nil)
+		el := p.Query("input")
+		p.Focus("input", nil)
+
+		kb.Press("Shift+KeyA", nil)
+		kb.Press("Shift+b", nil)
+		kb.Press("C", nil)
+		kb.Press("d", nil)
+		require.Equal(t, "AbCd", el.InputValue(nil))
+
+		metaKey := "Control"
+		if runtime.GOOS == "darwin" {
+			metaKey = "Meta"
+		}
+		kb.Press(metaKey+"+A", nil)
+		kb.Press("Delete", nil)
+		assert.Equal(t, "", el.InputValue(nil))
+	})
+
 	t.Run("newline", func(t *testing.T) {
 		p := tb.NewPage(nil)
 		cp, ok := p.(*common.Page)
@@ -60,5 +87,34 @@ func TestKeyboardPress(t *testing.T) {
 		kb.Press("Enter", nil)
 		kb.Type("World!", nil)
 		assert.Equal(t, "Hello\n\nWorld!", el.InputValue(nil))
+	})
+
+	// Replicates the test from https://playwright.dev/docs/api/class-keyboard
+	t.Run("selection", func(t *testing.T) {
+		t.Skip("FIXME") // See https://github.com/grafana/xk6-browser/issues/284
+		p := tb.NewPage(nil)
+		cp, ok := p.(*common.Page)
+		require.True(t, ok)
+		kb := cp.Keyboard
+
+		p.SetContent(`<input>`, nil)
+		el := p.Query("input")
+		p.Focus("input", nil)
+
+		kb.Type("Hello World!", nil)
+		require.Equal(t, "Hello World!", el.InputValue(nil))
+
+		kb.Press("ArrowLeft", nil)
+		// Should hold the key until Up() is called.
+		kb.Down("Shift")
+		for i := 0; i < len(" World"); i++ {
+			kb.Press("ArrowLeft", nil)
+		}
+		// Should release the key but the selection should remain active.
+		kb.Up("Shift")
+		// Should delete the selection.
+		kb.Press("Backspace", nil)
+
+		assert.Equal(t, "Hello!", el.InputValue(nil))
 	})
 }

